app/controllers/data.js

```diff
@@ -49,242 +49,233 @@ var DataController = Ember.ArrayController.extend({
-        }
-        else {
+        } else {
           return;
         }
       }
 
       if (this.get('showJSONInput')) {
         if (this.get('entityText') && this.get('entityText').length > 0) {
-          var entity;
           try {
-            entity = JSON.parse(this.get('entityText'));
-          }
-          catch (e) {
-            this.set('jsonError', 'Error parsing JSON: ' + e);
+            JSON.parse(this.get('entityText'));
+          } catch (e) {
+            this.set('jsonError', `Error parsing JSON: ${e}`);
             return;
           }
 
           if (this.get('showQueryInput')) {
             commandText += ', ';
           }
           commandText += this.get('entityText');
-        }
-        else {
+        } else {
           return;
         }
       }
 
-      var option, optionEnabled, optionInputValue;
-      var optionsArray = this.get('selectedCommandOptions');
-      var options = {};
-      for (var i=0; i < optionsArray.length; i++) {
+      let option; let optionEnabled; let
+        optionInputValue;
+      const optionsArray = this.get('selectedCommandOptions');
+      const options = {};
+      for (let i = 0; i < optionsArray.length; i += 1) {
         option = optionsArray[i];
-        optionEnabled = this.get(option + 'Option');
+        optionEnabled = this.get(`${option}Option`);
         if (optionEnabled) {
-          optionInputValue = this.get(option + 'OptionInput');
+          optionInputValue = this.get(`${option}OptionInput`);
           if (optionInputValue) {
             try {
               optionInputValue = JSON.parse(optionInputValue);
-            }
-            catch (e) {}
+            } catch (e) { /* empty */ }
             options[option] = optionInputValue;
-          }
-          else {
+          } else {
             options[option] = true;
           }
         }
       }
 
-      commandText += ', ' + JSON.stringify(options);
+      commandText += `, ${JSON.stringify(options)}`;
       commandText += ', function(err, result) {\n  if (err) {\n    // handle error here\n  }\n  else {\n    // handle success here\n  }\n  response.complete();\n});';
 
       this.set('commandCode', commandText);
     },
-    showCreateCollectionModal: function() {
+    showCreateCollectionModal() {
       this.set('newCollectionName', null);
     },
-    createCollection: function() {
-      var _this = this;
+    createCollection() {
+      const _this = this;
       if (this.get('newCollectionName') && this.get('newCollectionName').length > 0) {
-        return Ember.$.ajax('http://localhost:2845/collectionAccess/' + _this.get('newCollectionName') + '/collectionExists', { type: 'POST' , statusCode: { 200: function(data) {
-          if (data.exists) {
-            return _this.showError('A collection with that name already exists');
-          }
+        return Ember.$.ajax(`http://localhost:2845/collectionAccess/${_this.get('newCollectionName')}/collectionExists`, { type: 'POST',
```

Comment:
I think according to the style guide, this should be formatted a little differently https://github.com/Kinvey/style-guides/blob/master/backend/es6.md#functions--signature-invocation-indentation

Personally, I think it will be clearest by doing
```suggestion
        return Ember.$.ajax(`http://localhost:2845/collectionAccess/${_this.get('newCollectionName')}/collectionExists`, {
          type: 'POST',
          statusCode: {
            200(data) {
              ...
```

Reply:
Fixed. Thank you.